### DISCUSSION TITLE
react-use-cart

### DISCUSSION THREAD

**Discussion by bexruzlab** [npm/feedback#991] [General]
Hi everyone, I was gonna apply react-use-cart package to my project. while using usecart, called additem method of usecart. Then observed err that "Cannot read properties of undefined (reading 'id')" in my opinion, I always use mongoDB and every created document mongoDB provides uniqual _id. UseCart package accepts only "id" not "_id". What can be done about it ?

**Comment by ljharb** [comment]
File an issue with that package; it’s got nothing to do with npm.
